### PR TITLE
validate structure

### PR DESCRIPTION
**Pull request by sikhote**
Checks the output into the `dist` folder for changes.

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (mhewson) [inline .prettierrc.json:1]: what would be cool would be if this was stored someplace like satchel and could be shared via climbers to other microsites

File: README.md [hunk 2]
```diff
@@ -288,3 +288,7 @@ Found in `style-dictionary/transforms`
 See API for [creating a transform](https://amzn.github.io/style-dictionary/#/api?id=registertransform)
 
 See [transform docs](https://amzn.github.io/style-dictionary/#/transforms).
+
```
Comment (mhewson) [inline README.md:291]: any easy way to pass a bool into the build to validate that changes happened purposely - then automate the rebuild of validate-structure?

**Review by mhewson — APPROVED** [review]

File: .prettierrc.json (hunk 1 above)
Comment (sikhote) [reply .prettierrc.json:1]: Agreed, we could have an REI global one and everyone could opt in:
https://prettier.io/docs/en/configuration.html#sharing-configurations

File: README.md (hunk 2 above)
Comment (sikhote) [reply README.md:291]: Added in!

Comment (sikhote) [reply README.md:291]: Running `npm run validate -- --update` will do the trick

**Pull request merged**